doc/en/components/grids/_shared/export-excel.md

```diff
@@ -0,0 +1,234 @@
+</p>
+
+## {Platform} Excel Exporter Example
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz"
+           iframe-src="{environment:demosBaseUrl}/services/export-excel-sample-1" alt="Blazor Excel Exporter Example">
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height: 350px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz"
+           iframe-src="{environment:demosBaseUrl}/services/export-excel-tree-grid-sample" alt="Blazor Excel Exporter Example">
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height: 600px;" 
+        data-demos-base-url="{environment:demosBaseUrl}"
+        explicit-editor="stackblitz"
+        iframe-src="{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-excel-export" alt="Blazor Excel Exporter Example">
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
+
+
+## Exporting {Component}'s Data
+
+To start using the IgniteUI Excel Exporter first import the [`ExcelExporterService`]({environment:demosBaseUrl}/classes/excelexporterservice.html) in the app.module.ts file and add the service to the `providers` array:
+
+```razor
+// app.module.ts
+import { ExcelExporterService } from 'igniteui-{Platform}';
+
+@NgModule({
+  providers: [ ExcelExporterService ]
+})
+
+export class AppModule {}
+```
+
+> [!Note]
+> In v12.2.1 and later, the exporter services are provided in root, which means you no longer need to declare them in the AppModule providers.
+
+> [!NOTE]
+> The Excel Exporter service has a peer dependency on the JSZip library. The JSZip library should be installed when using the Excel Exporter.
+
+To initiate an export process you may use the handler of a button in your component's template.
+
+```html
+<{ComponentTitle} #{ComponentTitle} [data]="localData" [autoGenerate]="true"></{ComponentTitle}>
+<button (click)="exportButtonHandler()">Export {ComponentTitle} to Excel</button>
+```
+
+You may access the exporter service by defining an argument of type [`ExcelExporterService`]({environment:demosBaseUrl}/classes/excelexporterservice.html) in the component's constructor and the {Platform} framework will provide an instance of the service. To export some data in MS Excel format you need to invoke the exporter service's [`export`]({environment:demosBaseUrl}/classes/excelexporterservice.html#export) method and pass the {ComponentTitle} component as first argument.
```

Comment:
remove all hard-code links and just use API members (in camelcase) and wrap them in backticks, e.g. ``` `ExcelExporterService` ``` ``` `Export` ```